lib/active_admin/resource_controller/data_access.rb

```diff
@@ -250,6 +250,8 @@ def current_scope
       end
 
       def apply_pagination(chain)
+        # skip pagination if CSV format was requested
```

Comment:
@deivid-rodriguez, can you please clarify 

# return chain if params["format"] == "csv"

 so it should be commented out or not  ?

Reply:
Totally my bad. It should be uncommented. I think I left it commented out unintentionally while trying to write a failing test for this issue. I will fix it.

Reply:
I updated this!